app/script/calling/entities/FlowEntity.js

```diff
@@ -929,18 +951,17 @@ z.calling.entities.FlowEntity = class FlowEntity {
   }
 
   /**
-   * Check for relay candidate among given ICE candidates
+   * Counts the number of relay candidate in a list of ICE candidates.
    *
    * @private
    * @param {Array<string>} iceCandidates - ICE candidate strings from SDP
-   * @returns {boolean} True if relay candidate found
+   * @returns {number} The number of found relay ICE candidates
    */
-  _containsRelayCandidate(iceCandidates) {
-    for (const iceCandidate of iceCandidates) {
-      if (iceCandidate.toLowerCase().includes('relay')) {
-        return true;
-      }
-    }
+  _getNumberOfRelayCandidates(iceCandidates) {
+    return iceCandidates.reduce((count, iceCandidate) => {
+      const isRelay = iceCandidate.toLowerCase().includes('relay');
+      return isRelay ? count + 1 : count;
+    }, 0);
```

Comment:
Do we really need a reduce here or does `Array.filter().length` do the trick as well?

````
return iceCandidates.filter(iceCandidate => iceCandidate.toLowerCase().includes('relay')).length;
````